paasta_tools/cli/cmds/mesh_status.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import concurrent.futures
+from typing import List
+from typing import Sequence
+from typing import Tuple
+
+from paasta_tools.api.client import get_paasta_oapi_client
+from paasta_tools.cli.cmds.status import add_instance_filter_arguments
+from paasta_tools.cli.cmds.status import apply_args_filters
+from paasta_tools.cli.cmds.status import get_envoy_status_human
+from paasta_tools.cli.cmds.status import get_smartstack_status_human
+from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import load_system_paasta_config
+from paasta_tools.utils import PaastaColors
+from paasta_tools.utils import SystemPaastaConfig
+
+
+def add_subparser(subparsers) -> None:
+    mesh_status_parser = subparsers.add_parser(
+        "mesh-status",
+        help="Display the mesh status of a PaaSTA service.",
+        description=(
+            "'paasta mesh-status' queries the PaaSTA API in order to report "
+            "on the health of a PaaSTA service in the mesh."
+        ),
+    )
+    add_instance_filter_arguments(mesh_status_parser)
+    mesh_status_parser.add_argument(
+        "-v",
+        "--verbose",
+        action="count",
+        dest="verbose",
+        default=0,
+        help=(
+            "Prints the health of individual backends in the mesh, instead of "
+            "the overall health"
+        ),
+    )
+    mesh_status_parser.add_argument(
```

Comment:
Is this used?

Reply:
It is used implicitly. `apply_args_filters` requires it. I've added a comment to clarify.